gamla/higher_order_test.py

```diff
@@ -85,14 +85,14 @@ def set_item(key: str, value):
     def f(x):
         return x
 
-    assert (
-        higher_order.persistent_cache(
-            get_item,
-            set_item,
-            functional.make_hashed_call_key,
-        )(f)("something")
-        == "something"
-    )
+    cached_function = higher_order.persistent_cache(
+        get_item, set_item, functional.make_hashed_call_key
+    )(f)
+
+    result_set = cached_function("something")
```

Comment:
no need to set a variable here, you can just call `cached_function("something")`